modules/ROOT/pages/database-administration/aliases/manage-aliases-standard-databases.adoc

```diff
@@ -696,7 +696,7 @@ ALTER ALIAS `no-alias` IF EXISTS SET DATABASE TARGET `northwind-graph-2021`
 == Delete database aliases
 
 Both local and remote database aliases can be deleted using the `DROP ALIAS` command.
-The required privileges are described in the link:{neo4j-docs-base-uri}/cypher-manual/{page-version}/administration/access-control/dbms-administration/#access-control-dbms-administration-alias-management[Cypher Manual -> Access control -> DBMS administration].
+The required privileges are described in the xref:authentication-authorization/dbms-administration.adoc#access-control-dbms-administration-alias-management[Cypher Manual -> Access control -> DBMS administration].
```

Comment:
if the link no longer points to Cypher Manual, should the link text also be updated?
```suggestion
The required privileges are described in the xref:authentication-authorization/dbms-administration.adoc#access-control-dbms-administration-alias-management[Access control -> DBMS administration].
```